### PR TITLE
Add entrypoint to remove all references of an image

### PR DESCRIPTION
**Pull request by querti**

### CODE REVIEW

File: pubtools/_quay/remove_image.py [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+        umb_urls=umb_urls,
+        umb_cert=umb_cert,
+        umb_client_key=umb_client_key,
+        umb_ca_cert=umb_ca_cert,
+    )
+
+    sig_remover = SignatureRemover()
+    sig_remover.set_quay_client(quay_client)
+    for reference in references_to_remove:
+        sig_remover.remove_tag_signatures(
```
Comment (emilyzheng) [inline pubtools/_quay/remove_image.py:288]: It can't remove signatures of already untagged images.

Comment (querti) [reply pubtools/_quay/remove_image.py]: Good point, changed the order.

**Review by emilyzheng — APPROVED** [review]

File: pubtools/_quay/remove_image.py [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+def construct_kwargs(args):
+    """
+    Construct a kwargs dictionary based on the entered command line arguments.
+
+    Args:
+        args (argparse.Namespace):
+            Parsed command line arguments.
+
+    Returns (dict):
+        Keyword arguments for the 'remove_repository' function.
```
Comment (chandwanitulsi) [inline pubtools/_quay/remove_image.py:97]:
```suggestion
        Keyword arguments for the 'remove_images' function.
```

**Review by chandwanitulsi — COMMENTED** [review]
LGTM, Minor suggestions

File: pubtools/_quay/remove_image.py [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+        reference (str):
+            Images specified by tags to remove. Comma separated values.
+        quay_api_token (str):
+            OAuth token for authentication of Quay REST API.
+        quay_user (str):
+            Quay username for Docker HTTP API.
+        quay_password (str):
+            Quay password for Docker HTTP API.
+        pyxis_server (str):
+            Pyxis service hostname:
```
[the remark this answers is not in the document]
Comment (querti) [reply pubtools/_quay/remove_image.py]: Fixed

File: pubtools/_quay/remove_image.py (hunk 2 above)
Comment (querti) [reply pubtools/_quay/remove_image.py]: Fixed

**Comment by querti** [comment]
After fixing minor errors, I'm assuming that Tulsi's comment is a +1

**Pull request merged**